### DISCUSSION TITLE
解密后乱码

### DISCUSSION THREAD

**Discussion by xiancaoro** [guofei9987/blind_watermark#98] [Q&A] [answered]
使用提供的样图运行了几次
除开第一次成功解密之外，其余解密均会乱码😢，请问这种情况需要怎么处理呢🙏

**Comment by dusty-cjh** [comment, marked as answer]
发下你的测试代码

**Comment by galing0** [comment]
![image](https://github.com/guofei9987/blind_watermark/assets/170221754/3e0dd588-56e2-4ec0-94ed-30629f6aaa37)
我的代码跑起来也是乱码了，可以帮忙解答一下吗？其他的测试都不会，唯独这个没有将图片保存到本地的会报错

**Comment by Frankhao12138** [comment]
我的情况是图片使用微信传输并且没有发送原图之后再去提取就是乱码了，原始图片不会
